Sources/SimpleCurrencyConverter/DependencyInjection/DependencyContainer.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public final class DependencyContainer: DependencyProtocol {
+    
+    public static let shared = DependencyContainer()
+    
+    private init() {}
+    
+    var dependencies: [String: Any] = [:]
```

Comment:
Does `dependencies` is modified in an other class/struct?
If not, it should be `private(set) var dependencies: [String: Any] = [:]`